File: src/main/java/toker/panel/controller/rest/LanguageController.java
package toker.panel.controller.rest;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;
import toker.panel.entity.Language;
import toker.panel.repository.LanguageRepository;

import java.util.List;

@RestController
public class LanguageController {

    private LanguageRepository languageRepo;

    public LanguageController(LanguageRepository languageRepo) {
        this.languageRepo = languageRepo;
    }

    @GetMapping("/api/languages")
    public List<Language> languages() {
        return languageRepo.findAll();
    }
}

File: src/main/java/toker/panel/controller/rest/LanguageController.kt
package toker.panel.controller.rest

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController
import toker.panel.entity.Language
import toker.panel.repository.LanguageRepository

@RestController
class LanguageController(private val languageRepo: LanguageRepository) {
    @GetMapping("/api/languages")
    fun languages(): List<Language> {
        return languageRepo.findAll()
    }
}